model/src/main/kotlin/ScannerRun.kt

```diff
@@ -192,6 +200,8 @@ data class ScannerRun(
 
         val scanResultsByPath = resolutionResult.getKnownProvenancesWithoutVcsPath().mapValues { (_, provenance) ->
             scanResultsByProvenance[provenance].orEmpty()
+        }.mapValues { scanResults ->
```

Comment:
Could you use `(_ , scanResults) ->` instead of just `scanResults ->` ?

Reply:
This is `mapValues`, there is only one parameter.

Reply:
Isn't the value a tuple `(path, scanResults)` so it can be decomposed ?